src/Config.php

```diff
@@ -17,4 +17,19 @@ public function getTimeout(): int
     {
         return (int) $this->getValue(['parameters', 'timeout'], null);
     }
+
+    public function getToken(): string
+    {
+        return $this->getValue(['parameters', '#token']);
```

Comment:
Je to takto OK s tym tokenom? A budem ho tam vkladat zasifrovany?

Reply:
jo